### DISCUSSION TITLE
.next folder not generating on mac m1 pro

### DISCUSSION THREAD

**Discussion by kenn7575** [vercel/next.js#73407] [Help]
### Link to the code that reproduces this issue

https://github.com/kenn7575/plain_nextjs_starter_project

### To Reproduce

1. run ```npx create-next-app@latest```
2.  run ```npm run dev``` or ```npx next dev```

### Current vs. Expected behavior

After creating a new Next.js 15 project, I can't build the project to dev nor prod. Nothing except ```trace``` file is being generated inside .next folder, and the build process fails, stating that modules cannot be found. This happens on my MacBook M1 Pro with the latest stable versions of Node.js and npm installed. (node: 22.11.0, npm: 10.9.0)

### Provide environment information

```bash
Operating system:
MacOS Version 15.0 (24A335)

Computer:
MacBook Pro with M1 Pro arm chip

Binaries:
Node: 22.11.0
npm: 10.9.0 

NextJS version: 15.0.3
```

### Which area(s) are affected? (Select all that apply)

Developer Experience, Module Resolution, Output (export/standalone), Runtime, Script (next/script), Turbopack, Webpack

### Which stage(s) are affected? (Select all that apply)

next dev (local), next build (local), next start (local)

**Comment by icyJoseph** [comment]
I can't reproduce. With or without turbo.

```console
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 24.1.0: Thu Oct 10 21:03:15 PDT 2024; root:xnu-11215.41.3~2/RELEASE_ARM64_T6000
  Available memory (MB): 16384
  Available CPU cores: 8
Binaries:
  Node: 22.11.0
  npm: 10.9.0
  Yarn: N/A
  pnpm: 9.6.0
Relevant Packages:
  next: 15.0.3 // Latest available version is detected (15.0.3).
  eslint-config-next: 15.0.3
  react: 19.0.0-rc-66855b96-20241106
  react-dom: 19.0.0-rc-66855b96-20241106
  typescript: 5.7.2
Next.js Config:
  output: N/A
```

Most likely, you have not given permissions to npm, or something like that?
